### PR TITLE
Move code to 'firmware' folder,

### PR DESCRIPTION
**Pull request by learn-more**
allow opening platformio from the root of the repository
Additionally, I deleted `Info-Orbs/icons` and re-used the existing `icons/jpg white/` files.

### ❗ Please note, merging this will cause merge conflicts in all existing open PR's ❗

### CODE REVIEW

**Review by brett-dot-tech — APPROVED** [review]
This looks good, I'm going to wait to merge it though for 2-3 weeks until most folks have got their orbs (there are currently many in the mail) as to not distrust their flashing, after which I will add overlays on the youtube to accommodate this, just merging this now will be more distruptive now than it would be later

**Comment by brett-dot-tech** [comment]
@learn-more Could you move this PR to the dev branch? Going to start making major codebase changes there then doing big version level merges to main. I'll merge this right away when you do

**Comment by learn-more** [comment]
> @learn-more Could you move this PR to the dev branch? Going to start making major codebase changes there then doing big version level merges to main. I'll merge this right away when you do

I re-created the PR,
and I added the github actions (building) for the 'dev' branch as well as PR's to the 'dev' branch.

**Comment by brett-dot-tech** [comment]
@learn-more one final question here before finally getting this merge; Any reason behind keeping the Info-Orbs file for images the assets opposed to nesting it under an assets folder under the firmware? Is this strictly an organizational choice or is it required to be in a separate folder for PIO to compile

**Comment by learn-more** [comment]
> @learn-more one final question here before finally getting this merge; Any reason behind keeping the Info-Orbs file for images the assets opposed to nesting it under an assets folder under the firmware? Is this strictly an organizational choice or is it required to be in a separate folder for PIO to compile

No reason, I simply reused the PNG files that were already present in the repo.

File: platformio.ini
Comment (learn-more) [inline platformio.ini:20]: @brett-dot-tech I have updated these to use the images that were already present in your repository (one level above the old `Info-orbs`)
Due to how this embedding works, the `icons.h` also needed updating to reflect these paths.

**Pull request merged**
